libcodechecker/logger.py

```diff
@@ -54,137 +40,126 @@ def debug_analyzer(self, msg, *args, **kwargs):
 
 logging.setLoggerClass(CCLogger)
 
-
-class CustomFormatter(logging.Formatter):
-    """
-    Custom formatter to print log level in case of ERROR, WARNING
-    or CRITICAL.
-    """
-
-    # Short format strings where only the time and the message is printed.
-    format_string_short = {
-        'info_fmt':  '[%(asctime)s] - %(message)s',
-        'error_fmt': '[%(levelname)s] [%(asctime)s] - %(message)s',
-        'debug_fmt': '[%(asctime)s] {%(name)s} [%(process)d] <%(thread)d> - '
-                     '%(filename)s:%(lineno)d %(funcName)s() - %(message)s'
+DEFAULT_LOG_CFG_FILE = os.path.join(os.path.dirname(__file__),
+                                    'logger.conf')
+
+
+# Default config wich can be used if reading log config from a
+# file fails.
+DEFAULT_LOG_CONFIG = '''{
+  "version": 1,
+  "disable_existing_loggers": false,
+  "formatters": {
+    "brief": {
+      "format": "[%(asctime)s][%(levelname)s] - %(message)s",
+      "datefmt": "%Y-%m-%d %H:%M"
+    },
+    "precise": {
+      "format": "[%(levelname)s][%(asctime)s] - %(message)s",
```

Comment:
This formatter is not precise! This is same as brief. Why don't you use the precise formatter style what you have written in the `config/logger.conf` file:
```
"format": "[%(levelname)s] [%(asctime)s] {%(name)s} [%(process)d] <%(thread)d> - %(filename)s:%(lineno)d %(funcName)s() - %(message)s"
```

Reply:
Use `\` to fix long lines:
```json
    "precise": {
      "format": "[%(levelname)s] [%(asctime)s] {%(name)s} [%(process)d] \
<%(thread)d> - %(filename)s:%(lineno)d %(funcName)s() - %(message)s",
      "datefmt": "%Y-%m-%d %H:%M"
    }
```